scss/_settings_placeholders.scss

```diff
@@ -4,6 +4,7 @@
 
 // Global placeholder settings
 $input-border-thickness: 1.5px;
+$input-vertical-padding: calc($spv-nudge - $input-border-thickness);
```

Comment:
`$input-vertical-padding` is already defined in `_settings_spacing.scss`:
https://github.com/canonical/vanilla-framework/blob/ade7ef61988d065ed93306c32eb0e6107c133594/scss/_settings_spacing.scss#L121


Your comment [here](https://github.com/canonical/vanilla-framework/pull/4954#issuecomment-1906123371) also suggests that this variable should be in `_settings_spacing.scss`, not `_settings_placeholders.scss`

I'm also a bit confused generally about this variable as shown in this version of the PR, as you are declaring it here but not consuming it. Is the intention to override the pre-existing value of `$input-vertical-padding` for other sass files that are compiled after this one?

Reply:
Updated. Moved the variables to spacing settings file for clarity.